scripts/debug-xcode.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -uo pipefail
+
+OUTPUT=$(mktemp)
+
+# Print the command being run
+echo "==== Running command: $* ====\n"
+
+# Run wrapped command, capture both output and status
+"$@" 2>&1 | tee "$OUTPUT"
+STATUS=${PIPESTATUS[0]}
+
+# Try to find the xcodebuild log path
+LOGFILE=$(grep -oE '/var/folders/.*/xcodebuild-[A-F0-9-]+\.log' "$OUTPUT" | tail -1 )
```

Comment:
I think it would be better to drop the `/var/folders/.*/` from the regex, to make it usable in situations other than CI, too, if needed.